### ISSUE TITLE
join my channel bug

### ISSUE THREAD

**Issue by Ahmetaaaa** [brblacky/lavamusic#101]
1.If you kick the bot from the voice channel
2. And if you go to another voice channel 
3. And if you use the play command
4. Bot wants you to join the voice channel he was kicked before

**Comment by brblacky** [comment]
#29 
Fixed https://github.com/brblacky/lavamusic/commit/9af279be486c517c1c36abccf6511c119b657f50

**Comment by brblacky** [comment]
I think this bug fixes

**Comment by Ahmetaaaa** [comment]
Hi, I can explain better with screenshots.
1- I used the play command. : https://ibb.co/S6qHz5m
2- I kicked the bot from the voice channel. : https://ibb.co/KqRpjxj
3- The bot sends us a message that it has been kicked out of the voice channel and an error occurs, it sends the message that the song playing.
4- Let's switch to another voice channel.
5- I switched to voice channel and used the play command but the error occurred. : https://ibb.co/3Ck5nb5
6- Bot wants me to be on the same voice channel as with the bot, but bot is not in a voice channel.

I am using the latest version.
